### ISSUE TITLE
textual description is not included in converted nuscene data?

### ISSUE THREAD

**Issue by pengzhenghao** [metadriverse/scenarionet#73]
![Screenshot from 2024-03-26 15-39-05](https://github.com/metadriverse/scenarionet/assets/22206995/f25d6e5b-572a-42f2-89ff-cc86f4376f47)

I found that the text description is not included

**Comment by pengzhenghao** [comment]
![Screenshot from 2024-03-26 15-46-43](https://github.com/metadriverse/scenarionet/assets/22206995/ac4bd0a3-ed02-405d-b2ec-6759444baa23)

As shown in this figure, the exact same scenario in the raw data and in scenarionet. the description is removed.

**Comment by QuanyiLi** [comment]
Yeah, we can have it in the converted scenarios like nuPlan. Set as an enhancement.
